code/modules/mob/living/brain/MMI.dm

```diff
@@ -57,7 +57,7 @@
 		newbrain.brainmob = null
 		brainmob.forceMove(src)
 		brainmob.container = src
-		var/fubar_brain = newbrain.brain_death && newbrain.suicided && brainmob.suiciding //brain is damaged beyond repair or from a suicider
+		var/fubar_brain = newbrain.suicided && brainmob.suiciding //brain is damaged beyond repair or from a suicider
```

Comment:
comment says or but code says both are required

Reply:
is this suppose to have a organ damage check in place of the lost brain check?

Reply:
Hm yeah || seems right here. The organ damage check is on the line below, fubar is the permanent features.